lms/djangoapps/lti_provider/migrations/0004_require_user_account.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.22 on 2023-11-06 09:47
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('lti_provider', '0003_auto_20161118_1040'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='lticonsumer',
+            name='require_user_account',
+            field=models.BooleanField(blank=True, default=False, help_text='When checked, the LTI content will load only for learners who have an account in this instance. This is required only for linking leaner accounts with the LTI consumer. See the Open edX LTI Provider documentation for more details.'),
```

Comment:
```suggestion
            field=models.BooleanField(blank=True, default=False, help_text='When checked, the LTI content will load only for learners who have an account in this instance. This is required only for linking learner accounts with the LTI consumer. See the Open edX LTI Provider documentation for more details.'),
```

Without this, Django complains about changes that are not reflected in a migration.